src/permissions/TypicalBotMaintainer.js

```diff
@@ -12,7 +12,7 @@ class TypicalBotCreator extends PermissionLevelStr {
     }
 
     check(guild, member) {
-        return guild.client.config.maintainers.includes(member.id);
+        return process.env.MAINTAINERS.includes(member.id);
```

Comment:
Does `"123","456"` even work in the .env file and return an array?

Reply:
No. That's a string method.

Reply:
Oh right. I'm not sure why I was thinking it was an array method.